src/index.js

```diff
@@ -0,0 +1,324 @@
+const path = require('path');
+const inquirer = require('inquirer');
+
+const Manager = require('./lib/Manager');
+const Engineer = require('./lib/Engineer');
+
+const Intern = require('./lib/Intern');
+const fs = require('fs');
+const generateHTML = (manager, engineers, interns) => {
+  const generateEngineerCard = engineers
+    .map((engineer) => {
+      return `<div class="card" style="width: 18rem">
+  <div class="card-body">
+    <h5 class="card-title">engineer</h5>
+
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">${engineer.id}</li>
+      <li class="list-group-item">email:${engineer.email}</li>
+    </ul>
+  </div>
+</div>`;
+    })
+    .join('');
+
+  const generateInternCard = interns
+    .map((intern) => {
+      `<div class="card" style="width: 18rem">
+  <div class="card-body">
+    <h3 class="card-title">${intern.name}</h3>
+    <h5 class="card-title">intern</h5>
+
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">${intern.id}</li>
+      <li class="list-group-item">email: ${intern.email}</li>
+      <li class="list-group-item">school name: ${intern.school}</li>
+    </ul>
+  </div>
+</div>`;
+    })
+    .join('');
+  return `<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <meta charset="UTF-8" />
+      <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link
+        href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css"
+        rel="stylesheet"
+        integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor"
+        crossorigin="anonymous"
+      />
+      <title>Document</title>
+    </head>
+  
+    <body>
+      <nav class="navbar bg-primary">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">manager</a>
+        </div>
+      </nav>
+ 
+<div class="card" style="width: 18rem">
+<div class="card-body">
+  <h3 class="card-title">${manager.name}</h3>
+  <h5 class="card-title">${manager.id}</h5>
+
+  <ul class="list-group list-group-flush">
+    <li class="list-group-item">ID:123</li>
+    <li class="list-group-item">${manager.email}</li>
+    <li class="list-group-item">office number:${manager.officeNumber}</li>
+  </ul>
+</div>
+</div>
+      <!-- manager -->
+  
+      <nav class="navbar bg-primary">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">engineer</a>
+        </div>
+      </nav>
+  ${generateEngineerCard}
```

Comment:
You need to call the function here and pass in the `engineers`
```suggestion
  ${generateEngineerCard(engineers)}
```